### ISSUE TITLE
Throw exception on missing middleware

### ISSUE THREAD

**Issue by slavafomin** [balderdashy/sails#2604]
Hello!

Right now, Sails silently ignores if «**http.middleware.order**» array contains references to missing middleware. I think it's a critical defect. It should throw exceptions instead.

#### Consider this scenario

Developer creates a middleware named «**someVeryImportantLogicToRun**» which can contain some required security or logging functionality. After that he adds it to the orders array, but accidentally makes a mistake in the name: «**someImportantLogicToRun**». By looking on the order array and by running the application he will not notice the issue until something bad happens.

I hope it makes sense. Thank you!

**Comment by CWyrtzen** [comment]
Hi @slavafomin is this still an issue with the latest release?

**Comment by sailsbot** [comment]
Thanks for posting, @slavafomin.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
